docs/example/vllm/build/ray_init.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -e
+subcommand=$1
+shift
+
+ray_port=6379
+ray_init_timeout=300
+
+case "$subcommand" in
+  worker)
+    ray_address=""
+    while [ $# -gt 0 ]; do
+      case "$1" in
+        --ray_address=*)
+          ray_address="${1#*=}"
+          ;;
+        --ray_port=*)
+          ray_port="${1#*=}"
+          ;;
+        *)
+          echo "unknown argument: $1"
+          exit 1
+      esac
+      shift
+    done
+
+    if [ -z "$ray_address" ]; then
+      echo "Error: Missing argument --ray_address"
+      exit 1
+    fi
+
+    for (( i=0; i < $ray_init_timeout; i+=5 )); do
+      ray start --address=$ray_address:$ray_port --block
+      if [ $? -eq 0 ]; then
```

Comment:
I think we should have a feature in LeaderWorkerSet to not create the workers sts until the leader is ready so that we don't need to do these kinds of checks. I created https://github.com/kubernetes-sigs/lws/issues/135 @liurupeng @kerthcet